src/index.js

```diff
@@ -1,14 +1,23 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { BrowserRouter as Router } from "react-router-dom";
 import App from "./App";
 import { makeServer } from "./server";
+import "./index.css";
+import { MenuProvider, VideoProvider } from "hooks";
```

Comment:
you can sort the imports to look clean, like all the local imports from different files at the bottom and the imports from dependencies at the top, just nitpicking